### PR TITLE
Add GLIMPSE (bio)

### PR DESCRIPTION
**Pull request by nh13**
Adding a new recipe for [GLIMPSE](https://odelaneau.github.io/GLIMPSE/).
----

TODO:
- [x] PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/glimpse-bio/build.sh
Comment (nh13) [inline recipes/glimpse-bio/build.sh:11]: TODO: need to install the binaries!

File: recipes/glimpse-bio/build.sh
Comment (nh13) [inline recipes/glimpse-bio/build.sh:15]:
```suggestion
# Install the binaries
```

File: recipes/glimpse-bio/build.sh
Comment (nh13) [inline recipes/glimpse-bio/build.sh:9]: ^ the makefiles from GLIMPSE are non-standard, reported to the GLIMPSE developers: https://github.com/odelaneau/GLIMPSE/pull/64

**Comment by nh13** [comment]
@BiocondaBot please review & merge

File: recipes/glimpse-bio/meta.yaml
Comment (bgruening) [inline recipes/glimpse-bio/meta.yaml:34]: Is boost-cpp enough? or do you need for example the python bindings as well?

File: recipes/glimpse-bio/meta.yaml
Comment (bgruening) [inline recipes/glimpse-bio/meta.yaml:32]: you can remove zlib, bzip2 and htslib I think from the run deps

File: recipes/glimpse-bio/meta.yaml
Comment (nh13) [reply recipes/glimpse-bio/meta.yaml:34]: There is no Python based on my understanding.

File: recipes/glimpse-bio/meta.yaml
Comment (nh13) [reply recipes/glimpse-bio/meta.yaml:32]: Happy to do so, but curious why so I can learn for my next recipe.

**Comment by nh13** [comment]
@bgruening comments addressed and I’ll wait for your approval.

File: recipes/glimpse-bio/meta.yaml
Comment (bgruening) [reply recipes/glimpse-bio/meta.yaml]: For libraries, we can define `run_exports`, so this is implicit: https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L13

It just cleans up recipes on your side and makes sure it gets really exported, every time, on the global ecosystem.

File: recipes/glimpse-bio/meta.yaml
Comment (bgruening) [reply recipes/glimpse-bio/meta.yaml:31]: then you can probably remove `boost`. `boost` includes additional dependencies __and__ `boost-cpp`. So I assume you only need boost-cpp.

Comment (nh13) [reply recipes/glimpse-bio/meta.yaml:29]: Done

**Comment by nh13** [comment]
@bgruening I have addressed (and resolved) all your comments.  Is this ready for approval?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by bgruening** [comment]
@nh13 yes, thanks very much for your contribution.

The package will be available in ~30min and the container in ~20min.

**Comment by nh13** [comment]
Thank-you!

**Pull request merged**
